pkgs/applications/editors/vscode/generic.nix

```diff
@@ -108,6 +111,18 @@ let
       )
     '';
 
+    # See https://github.com/NixOS/nixpkgs/issues/49643#issuecomment-873853897
+    postPatch = ''
+      # this is a fix for "save as root" functionality
+      packed="resources/app/node_modules.asar"
+      unpacked="resources/app/node_modules"
+      ${nodePackages.asar}/bin/asar extract "$packed" "$unpacked"
+      substituteInPlace $unpacked/sudo-prompt/index.js \
+        --replace "/usr/bin/pkexec" "/run/wrappers/bin/pkexec" \
+        --replace "/bin/bash" "${bash}/bin/bash"
```

Comment:
Do we need readline support? If so please use bashInteractive.

Reply:
Could you suggest the necessary change? I have, ehm, zero Nix experience.

Reply:
Pshirshov said he tested it so I assume we don't